odoo/addons/base/models/ir_asset.py

```diff
@@ -210,6 +210,8 @@ def _process_path(self, bundle, directive, target, path_def, asset_paths, seen,
             return
         if can_aggregate(path_def):
             paths = self._get_paths(path_def, installed)
+            if not paths:
```

Comment:
```suggestion
            if paths is None:
```
Good to use this method since already known paths is None https://github.com/odoo/odoo/blob/b2c83492c0d73244b22dc0fe7a971c9ac8c8915d/odoo/addons/base/models/ir_asset.py#L339